src/PKSim.Core/Snapshots/Mappers/ClassificationSnapshotTask.cs

```diff
@@ -56,8 +56,11 @@ public ClassificationSnapshotTask(ClassificationMapper classificationMapper)
          snapshot.Classifiables?.Each(snapshotClassifiable =>
          {
             var subject = subjects.FindByName(snapshotClassifiable);
-            var classifiable = project.GetOrCreateClassifiableFor<TClassifiable, TSubject>(subject);
-            classifiable.Parent = classification;
+            if (subject != null)
```

Comment:
This happens if the simulation for example was not loaded into project for some unknown reason (a crash). Then trying to sort that simulation in the folder structure would crash